docs/pages/User/GettingStarted/cleanup.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Clean-up 
+weight: 5
+---
+
+
+## Clean-up
+
+You can remove all the deployed examples with:
```

Comment:
There are two command here. I would tell the reader what they do (i.e. why we have to run them both).